sha256sums.txt

```diff
@@ -13,6 +13,7 @@ fdcaedc1fb19a63d7d875b0105b0a5b496dd0989330d454a42bcb401fa5454ec  furl-2.0.0.tar
 87ee3f74ba3ea544e683a5a22e7e34f4d1cf3ad34414b5f3858becf00facf1d6  Mako-1.0.7-py3-none-any.whl
 4e02fde57bd4abb5ec400181e4c314f56ac3e49ba4fb8b0d50bba18cb27d25ae  Mako-1.0.7.tar.gz
 6a7078a2fb4406458d6ae3579e4eb01a9bdc0a9a0686a28fa50c19a039e3fcb8  MarkupSafe-1.0-cp35-cp35m-linux_x86_64.whl
+3c9bf8fb4c3cf7dd11fd465132156d4c3cddb926d39bdbd0f0bf5920fd8009a4  MarkupSafe-1.0-cp37-cp37m-linux_x86_64.whl
```

Comment:
optional: add the linux_x86_64 Python 3.7 wheel for PyYAML also? (optional because this PR does specifically say "for the client" so we can do in two PRs if you prefer. since we also need to update wheels for buster for e.g. the proxy adding that one wheel would be nice because then we are good to go with respect to buster and our pip mirror), see #82

Reply:
Done, and the PR updated.

Reply:
kewl thanks!